php-ts-mode.el

```diff
@@ -77,6 +78,7 @@
        ;; "compound_statement" contains the body of many statements.
        ;; For example function_definition, foreach_statement, etc.
        ((parent-is "compound_statement") parent-bol ,offset)
+       ((parent-is "method_declaration") parent-bol 0)
```

Comment:
Would you be able to add a test for this case?
We have very basic suite here: https://github.com/emacs-php/php-ts-mode/blob/master/tests/php-ts-mode-resources/indent.erts

It would be great to increase the coverage while we fix issues.